src/info.c

```diff
@@ -30,8 +30,44 @@ void print_update_conf_info()
 	printf("Content URL:       %s\n", content_url);
 }
 
+static void print_help(void)
+{
+	fprintf(stderr, "Usage:\n");
+	fprintf(stderr, "   swupd info [OPTION...]\n\n");
+
+	global_print_help();
+	fprintf(stderr, "\n");
+}
+
+static const struct global_options opts = {
+	NULL,
+	0,
+	NULL,
+	print_help,
+};
+
+static bool parse_options(int argc, char **argv)
+{
+	int ind = global_parse_options(argc, argv, &opts);
+
+	if (ind < 0) {
+		return false;
+	}
+
+	if (argc > ind) {
+		fprintf(stderr, "Error: unexpected arguments\n\n");
```

Comment:
you can use the `error()` from the logger here, we are already changing that, so it is time to start using that.